pkg/webui/console/components/webhook-form/index.js

```diff
@@ -151,6 +153,7 @@ export default class WebhookForm extends Component {
           component={Input}
           required
         />
+        <Form.Field name="downlink_api_key" title={m.downlinkAPIKey} component={Input} code />
```

Comment:
Does it make sense to add a small description to this field? Can't tell if its use is obvious enough.